app/controllers/transcriptions_controller.rb

```diff
@@ -80,6 +107,27 @@ def allowed_filters
     [:id, :workflow_id, :group_id, :flagged, :status]
   end
 
+  def status_has_changed?
+    update_attrs.dig('status') != @transcription.status
+  end
+
+  def update_transcription_exports
+    if update_attrs['status'] == 'approved'
```

Comment:
There's an `approve?` method for this already, so you can just say `if approve?`, maybe it needs a rename.

Reply:
Ah, definitely. Maybe that was added after I wrote the code for this method? Or I just missed it, who knows. Either way, agreed.

Reply:
agree that it's ambiguous, change it to whatever you think is better